app/controllers/uploads_controller.rb

```diff
@@ -87,12 +87,18 @@ def call_load
     file = @upload.upload_file.tempfile
     skip_lines = @upload.skip_lines.try(:to_i)
     col_sep = @upload.col_sep
+
+    CrosswalkIssue.delete_all if [Crosswalk, IpedsHd, Weam].include?(klass)
```

Comment:
This has to be done before `klass.load` it attempts to delete the contents of the uploading model table, which throws an exception because we added foreign keys to `crosswalk_issues`.